src/services/messengerSync/utils.js

```diff
@@ -0,0 +1,151 @@
+const { userModel, displayName } = require('../user/model');
+const { schoolModel } = require('../school/model');
+const roleModel = require('../role/model');
+const { courseModel } = require('../user-group/model');
+const { teamsModel } = require('../teams/model');
+const { Configuration } = require('@schul-cloud/commons');
```

Comment:
![Codacy](https://app.codacy.com/assets/images/favicon.png) Issue found: [`@schul-cloud/commons` import should occur before import of `../user/model`](https://app.codacy.com/manual/schul-cloud/schulcloud-server/pullRequest?prid=5203703)